packages/vuetify/src/locale/ar.ts

```diff
@@ -54,16 +54,16 @@ export default {
   },
   pagination: {
     ariaLabel: {
-      wrapper: 'ترقيم الصفحات الملاحة',
+      wrapper: 'الإنتقال بين الصفحات',
       next: 'الصفحة التالية',
       previous: 'الصفحة السابقة',
-      page: '{0} انتقل إلى صفحة',
-      currentPage: '{0} الصفحة الحالية ، الصفحة',
+      page: '{0} انتقل إلى الصفحة',
+      currentPage: '{0} الصفحة الحالية هي رقم',
     },
   },
   rating: {
     ariaLabel: {
-      icon: 'Rating {0} of {1}',
+      icon: 'القييم {0} من أصل {1}',
```

Comment:
I am about to suggest to replace من أصل to simply على but i am not sure if it would make always sense. Let me know what you think as it's shorter

Reply:
I was aware that some translations were kinda verbose, but could not shorten them without introducing a possibility for confusion.

My personal priority is clarity, so I'm gonna stand with my change. Moreover, this is an aria label, so it's not like it's gonna clutter the screen or anything, I can even say that in this case specifically shorter is not better at all.

Reply:
How about simply using the word من? i don't think it would sound confusing as you prefixed the phrase with التقييم. I find the term من اصل here a bit oddly used.

Reply:
Sure, will commit this change momentarily.